mobile/src/main/java/rs/readahead/washington/mobile/views/activity/AudioRecordActivity2.java

```diff
@@ -468,4 +489,14 @@ private void updateStorageSpaceLeft(long memoryLeft) {
             freeSpace.setText(getString(R.string.days_hours_and_space_left, days, hours, spaceLeft));
         }
     }
+
+    private void scheduleFileUpload(MediaFile mediaFile) {
+        if (Preferences.isAutoUploadEnabled()) {
+            List<MediaFile> upload = new ArrayList<>();
+            upload.add(mediaFile);
+            uploadPresenter.scheduleUploadMediaFiles(upload);
```

Comment:
You can use `Collections.singletonList(mediaFile)` here or even better - change presenter method signature to accept single `MediaFile` as it is never intended to be used for list of media files.

Reply:
This is temporary and it will be changed soon. I'm working on scheduling uploads using this presenter on the user's request from the gallery.

Reply:
So, presenter will accept List<MediaFile>? In that case, I suggest that Collections call for single item list.